hack/make/.binary

```diff
@@ -70,7 +70,7 @@ hash_files() {
 
 	# -buildmode=pie is not supported on Windows and Linux on mips and riscv64.
```

Comment:
Can you update the comment here?

Reply:
That Go source link you shared would be a great candidate for this comment, IMO :eyes:

Reply:
applied both suggestions.
btw that go source file shows that `riscv64` supports PIE.
so `.binary`  forces people who build docker on riscv64 on newer go versions to use non-PIE. something to keep an eye on.
I don't have `riscv64` HW to test/confirm/deny that though.